source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,42 @@ The global vulnerability database is created automatically, currently pulling da
+In any of these cases, the packages and the operating system that were already scanned will not be re-scanned, only the new ones will do and alerts are going to be generated.
+If the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires all packages as well as the operating system will be re-scanned.
+Reducing the times that the `Full scans` are triggered improves the usage of the system resources and increase the performance avoiding repeat alerts of already notified vulnerabilities during a configurable time.
 
-- Baseline: This scan type will be triggered the first time after Vulnerability Detector is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
-- Full scan: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+For more details we have then three different scan types that are described below.
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
+- `Partial scan`: Only new packages are scanned and the new vulnerabilities detected are alerted together with deleted or updated packages.
 
 There are few considerations that arise from this behavior:
 
-- The user can not trigger a full scan manually. The only option is to disable and enable Vulnerability Detector to trigger the baseline scan. For that purpose, a service restart is needed.
-- The partial scans generate alerts for new packages, but they do not delete alerts for removed packages.
-- Partial scans can be triggered with a Manager restart.
+- The user can not trigger a `Full scan` manually. The only option is to disable and enable `Vulnerability Detector` to trigger the `Baseline` scan. For that purpose, a service restart is needed.
```

Comment:
Not sure whether this is totally true. We should suggest in any case the reduction of the `min_full_scan_interval`.

Reply:
With the new behavior, this workaround wouldn't do anything. 
Also, a `min_full_scan_interval` reduction would only trigger a full scan if the feeds were updated.

Reply:
This isn´t correct right now. I think it will be better to avoid mentioning the lack of functionalities and remove this paragraph

Reply:
I agree that would be better not to mention this.